modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/configuration/helper/FriendlyURLRedirectionConfigurationHelperImpl.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.friendly.url.internal.configuration.helper;
+
+import com.liferay.friendly.url.configuration.helper.FriendlyURLRedirectionConfigurationHelper;
+import com.liferay.friendly.url.internal.configuration.FriendlyURLRedirectionConfiguration;
+import com.liferay.friendly.url.internal.configuration.admin.service.FriendlyURLRedirectionManagedServiceFactory;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Daniel Mijarra
+ */
+@Component(
+	configurationPid = "com.liferay.friendly.url.internal.configuration.FriendlyURLRedirectionConfiguration",
+	service = FriendlyURLRedirectionConfigurationHelper.class
+)
+public class FriendlyURLRedirectionConfigurationHelperImpl
+	implements FriendlyURLRedirectionConfigurationHelper {
+
+	@Override
+	public String redirectionType(long companyId) {
```

Comment:
Sorry I didn't see this before, but shouldn't this be `getRedirectionType`?

Reply:
Yes, no problem

Reply:
@adolfopa I have added a new commit with the change, see: 2c8d3fff9b82c34d54083e3030594358ed451912